src/main/scala/learnshapeless/HLists.scala

```diff
@@ -78,20 +88,25 @@ object HLists extends App {
   object IsAustralianPoly extends DefaultIdentityMapping {
     implicit def isAustralia = at[Country](_ == Australia)
   }
-  def ex_poly_country = ???
+
+  def ex_poly_country: String :: Int :: Boolean :: HNil =
+    ex_poly.map(IsAustralianPoly)
```

Comment:
But I guess if you ask the attendees to spell out the type, you'd immediately know why.